BFS.py

```diff
@@ -52,52 +50,56 @@ def expand(board, node, boxes, path):
     ##UP##
     if board[node[0] - 1][node[1]] != WALL:                         #if UP is not a wall
         if board[node[0] - 1][node[1]] == BOX:                          #if UP is a box
-            if board[node[0] - 2][node[1]] != WALL:                         #if you can push the box UP
+            if board[node[0] - 2][node[1]] != WALL\
```

Comment:
is an array index out of bounds exception ever possible in this logic?

Reply:
nope